src/modeler/AutoRest.Swagger/Validation/SummaryDescriptionValidation.cs

```diff
@@ -33,7 +37,7 @@ public class SummaryDescriptionValidation: TypedRule<Operation>
 
         public override IEnumerable<ValidationMessage> GetValidationMessages(Operation operation, RuleContext context)
         {
-            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim()))
+            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.CurrentCultureIgnoreCase))
```

Comment:
Why did you decide to use culture aware case-insensitive comparison by `CurrentCultureIgnoreCase`.

We should use do `case-insensitive` + `culture-insensitive` comparison by using `StringComparison.OrdinalIgnoreCase`